.github/workflows/build-branch.yml

```diff
@@ -84,10 +84,8 @@ jobs:
 
   # create checks
   create-checks:
```

Comment:
ah but we want to keep the "always" for that one too :)

Reply:
Even in the foreign repo?

Reply:
Updated.